src/components/taskDetails/ProgressUpdateCard/ProgressUpdateCardPresentation.tsx

```diff
@@ -59,33 +66,97 @@ export default function ProgressUpdateCardPresentation({
                     <h3 className={styles['progress-update-card__title']}>
                         {titleToShow}
                     </h3>
-                    <Tooltip
-                        tooltipPosition={{ top: '-25px', right: '-2rem' }}
-                        content={tooltipString}
+                    <div
+                        className={
+                            styles['progress-update-card__details-container']
+                        }
```

Comment:
why are we adding this new div wrapper?

Reply:
The `progress-update-card__details-container` is a wrapper around the date and user profile image that ensures everything is styled consistently.

Reply:
Previously, this wrapper wasn't required and I see that only the avatar modifications inside the wrapper. Was this change made to address any issues with the previous style?

Reply:
The parent div has a justify-between property and currently contains only two children. Adding another child directly to the parent div would make the layout messy. To avoid this, I need to group the date and avatar into a single div, which will help maintain the layout's structure.